### PR TITLE
:book: Add METRO env variable to Equinix Metal quickstart

### PR DESCRIPTION
**Pull request by vasubabu**
**What this PR does / why we need it**:

This PR adds a couple lines to the quick start guide to take advantage of automated, interconnected bare metal across our global metros.

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #

### CODE REVIEW

File: docs/book/src/user/quick-start.md [hunk 1]
```diff
@@ -928,6 +928,7 @@ are also a few optional tunables if you'd like to change the OS or CIDRs used.
 export PROJECT_ID="2b59569f-10d1-49a6-a000-c2fb95a959a1"
 # The facility where you want your cluster to be provisioned
 export FACILITY="da11"
+export METRO="da"
```
Comment (killianmuldoon) [inline docs/book/src/user/quick-start.md:931]: Could you add  a comment explaining what this env variable is - like in the others in the list.

I think you also need to sign the CLA to pass that check.

**Review by killianmuldoon — COMMENTED** [review]
/ok-to-test

File: docs/book/src/user/quick-start.md (hunk 1 above)
Comment (vasubabu) [reply docs/book/src/user/quick-start.md:931]: METRO ENV variable can help to take advantage of automated, interconnected bare metal across our global metros.

Comment (killianmuldoon) [reply docs/book/src/user/quick-start.md:931]: Can you add this information inline in the markdown - just a simple comment stating what the env variable is used for is enough.

**Comment by fabriziopandini** [comment]
Please add some explanation in the PR summary and possibly make the PR title more self-explanatory (it will be used for generating release notes)

**Comment by killianmuldoon** [comment]
/retitle :book: Add METRO env variable to Equinix Metal quickstart

**Comment by killianmuldoon** [comment]
@fabriziopandini I think we don't really need anything in the PR summary - we merge a lot of PRs with either empty or outdated summaries and don't use them in the release notes.

Is this something you think should be changed going forward?

File: docs/book/src/user/quick-start.md [hunk 2]
```diff
@@ -928,6 +928,8 @@ are also a few optional tunables if you'd like to change the OS or CIDRs used.
 export PROJECT_ID="2b59569f-10d1-49a6-a000-c2fb95a959a1"
 # The facility where you want your cluster to be provisioned
 export FACILITY="da11"
```
Comment (killianmuldoon) [inline docs/book/src/user/quick-start.md:930]: /hold 

Should FACILITY be removed as part of this flow?

Ref from the Equinix Metal Provider README - https://github.com/kubernetes-sigs/cluster-api-provider-packet#ugrading-to-v07x

**Review by cprivitere — APPROVED** [review]
/lgtm

File: docs/book/src/user/quick-start.md (hunk 2 above)
Comment (cprivitere) [reply docs/book/src/user/quick-start.md:930]: Yes, it should be removed.

**Comment by cprivitere** [comment]
/remove-lgtm
pending the removal of the facility variable

File: docs/book/src/user/quick-start.md (hunk 2 above)
Comment (vasubabu) [reply docs/book/src/user/quick-start.md]: It's removed

**Review by fabriziopandini — COMMENTED** [review]
/lgtm
/approve

**Pull request closed without merging**